autoload/lsp/utils.vim

```diff
@@ -76,6 +76,17 @@ else
     endfunction
 endif
 
+if has('win32') || has('win64')
+    function! lsp#utils#normalize_uri(uri) abort
```

Comment:
add a comment to https://github.com/microsoft/language-server-protocol/pull/1019 for future reference.